newIDE/app/src/SceneEditor/index.js

```diff
@@ -559,6 +571,19 @@ export default class SceneEditor extends React.Component<Props, State> {
     this.updateToolbar();
   };
 
+  _addNewObjectAndInstance = (newObjectName: string) => {
```

Comment:
This method is not creating any new object, so let's name it `_onAddNewObjectInstance`